src/types/typeobject.rs

```diff
@@ -35,6 +38,29 @@ impl PyType {
         self.getattr(intern!(self.py(), "__qualname__"))?.extract()
     }
 
+    /// Gets the full name including the module of the `PyType`.
+    pub fn full_name(&self) -> PyResult<Cow<'_, str>> {
```

Comment:
I can see this replacing `name`, but on the other hand, this appears to be needless churn.

Reply:
I'd be quite happy to have `.name()` and `.qualname()` if we didn't mind a bit of churn to match CPython better

Reply:
I suppose we could call them `get_name` and `get_qualname` to avoid silent changes?

Reply:
Would work, but would also be unidiomatic. I did change the return type form `&str` to `String` which might already induce some breakage (to be compatible with a `Py2`-based implementation). Let me know if this (and the migration guide entry pushed here) seems sufficient for you. Otherwise, I'll amend the commit to use the `get_` prefixes.

Reply:
Yep that looks okay to me 👍